Client/core/CCore.cpp

```diff
@@ -1994,6 +2018,8 @@ void CCore::OnDeviceRestore()
 //
 void CCore::OnPreFxRender()
 {
+    CGraphics::GetSingleton().DrawEffekseerEffects();
```

Comment:
Internally **DrawEffekseerEffects** invokes **EnteringMTARenderZone** method which is quite expensive. I would be better to place **DrawEffekseerEffects** between the existing EnteringMTARenderZone/LeavingMTARenderZone call sites within the **CCore::OnPreFxRender** somehow.